environment/src/main/resources/schema/app/20220303161846_CB-15814_Option_to_create_without_load_balancer.sql

```diff
@@ -0,0 +1,14 @@
+-- // CB-15814 Add option
+-- Migration SQL that makes the change goes here.
+
+
+ALTER TABLE environment_network ADD IF NOT EXISTS loadBalancerStatus varchar(255);
```

Comment:
DEFAULT 'ENABLED' can be moved to here, NOT NULL also

example: ALTER TABLE mytable ADD COLUMN mycolumn character varying(50) DEFAULT 'whatever' NOT NULL;